media/js/app/taxonomy/taxonomy.js

```diff
@@ -442,31 +448,45 @@
 
             // split the url.
             var urls = value.split(',');
+            var flagArray = [];
```

Comment:
Good point. Maybe the isJson regex could be expanded to check for either the onomy url pattern or the test url pattern.